app/managers.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            "INSERT INTO actors (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self):
+        actor_cursor = self._connection.execute(
+            "SELECT * FROM actors"
+        )
+        return [Actor(*row) for row in actor_cursor]
+
+    def update(self, id_to_update: int,
+               new_first_name: str,
+               new_last_name: str) -> None:
+        self._connection.execute(
+            "UPDATE actors "
+            f"SET {new_first_name} = ?, {new_last_name} = ? "
```

Comment:
The SQL syntax in the `SET` clause is incorrect. It should be `SET first_name = ?, last_name = ?` instead of using the new names as column names.